WebApp/ISIS/autoreduce_webapp/reduction_variables/views.py

```diff
@@ -394,17 +397,17 @@ def run_confirmation(request, instrument=None):
         if len(new_variables) == 0:
             context_dictionary['error'] = 'No variables were found to be submitted.'
 
-		# User can choose whether to overwrite with the re-run or create new data
+        # User can choose whether to overwrite with the re-run or create new data
```

Comment:
one more tab here

Reply:
That's not a tab it's 8 spaces

Reply:
Yes just noticed that also. Apologies. For another day the PEP 8 may have something used of spaces between lines of codes

Reply:
Okay I'll setup PEP 8 in my IDE and ensure it passes when I make changes.